setup.py

```diff
@@ -2,15 +2,15 @@
 
 setup(
     name="hca-metadata-api",
-    version="1.0b23.dev1",
+    version="1.0b24.dev1",
```

Comment:
This version spec will be changed as part of the release process. PRs can't touch it.